packaging/rpm/microshift.spec

```diff
@@ -394,6 +394,7 @@ if [ $1 -eq 1 ]; then
 	# if crio was already started, restart it so it will catch /etc/crio/crio.conf.d/10-microshift.conf
 	systemctl is-active --quiet crio && systemctl restart --quiet crio || true
 fi
+systemctl enable --now crio-wipe
```

Comment:
For crio, we do not enable it explicitly, just note the dependency.
I think we might want to do the same for crio-wipe.
See https://github.com/openshift/microshift/blob/main/packaging/systemd/microshift.service#L3